docs/intro.md

```diff
@@ -6,6 +6,31 @@ sidebar_label: Godwoken Introduction
 ---
 import useBaseUrl from "@docusaurus/useBaseUrl";
 
+## Some Words of Caution
+We would like to mention that the current version available is the Godwoken beta. We still maintain various levels of control over the mainnet. 
+
+**Why Godwoken Beta?**
+We believe it is the responsible way to make Godwoken available while we keep up improving the system before the official release.
+Here is what you can do with Godwoken beta:
+
+| features | maintainer | developer |
+| :------ | :-------: | :-------: |
+| deploy contract | yes | trusted*  | 
+| deploy ERC20 proxy contract  | yes | no |
+| run full nodes  | yes | no |
+| run readonly nodes  | yes | no** |
+| launch challenges  | no | no |
+
+*It is only accessible to the developers on the whitelist. As requested by Nervos, we will maintain a whitelist of developers during the mainnet beta phase.
```

Comment:
```suggestion
* During the mainnet beta phase, the deploy contract feature is only open to developers on the allowed list. It is for emergencies, in case we need to upgrade the mainnet beta or coordinate with developers. However, developers or teams are welcome to participate in the mainnet beta.
```

Reply:
调整了下：During the mainnet beta phase, the deploy contract feature is only open to developers allowed on the list. We need to upgrade the mainnet beta or coordinate with developers in case of emergency. Developers or teams are welcome to participate in the mainnet beta anyways.